### ISSUE TITLE
Add Live Docs

### ISSUE THREAD

**Issue by Chetan11-dev** [pytube/pytube#1598]
**Is your feature request related to a problem? Please describe.**
Have  a Live usable version of PyTube as Docs 

**Describe the solution you'd like**
We have a usable Input Card to use PyTube and below it we have Code Examples as how to use that feature. Similar to Lodash Js (They use RunKit)
I created a Demo you may use at https://www.omkar.cloud/tools/get-youtube-thumbnail/

**Describe alternatives you've considered**
- I have not considered any other alternatives.

**Additional context**
I am Full Stack Developer and ready to provide Code for it's implementation.

**Comment by RONNCC** [comment]
@Chetan11-dev  would love a PR for it -> wonder perhaps  where/how it would be hosted

**Comment by Chetan11-dev** [comment]
@RONNCC 
I apologize for the delay in providing an update. I wanted to show you a live demonstration of how the documentation would look like, so I created documentation for a package I built `bose`, at end the documentation of pytube will look like https://www.omkar.cloud/bose/docs/introduction/.

I suggest following things:

- We can use Docusaurus as the documentation framework, documentation of pyyube will look similar to https://www.omkar.cloud/bose/.
- We can store all documentation in the "pytube" repository under the "docs/" folder.
- We can add a card above the documentation for the "search feature," "Channels," "Playlists," "Caption," and "Downloading Streams" to make it more interactive.
- We can host pytube docs at omkar.cloud/pytube/.

I am prepared to provide a pull request to migrate the current documentation to Docusaurus and include the card functionality as mentioned. This will make the "pytube" documentation more user-friendly and attractive.

Let me know if you have any questions.

**Comment by RONNCC** [comment]
Hey @Chetan11-dev 

Is it correct that, this change would then port docs over from a more 'open-source' friendly (https://readthedocs.org or Github Pages) doc structure to one owned by the omkar.cloud provider effectively?

**Comment by Chetan11-dev** [comment]
Yeah, we already have systems in place so we could host it. Is that fine?

**Comment by RONNCC** [comment]
Ah, perhaps it is harder to keep all related code in one place --> perhaps we could spruce up the readme.com perhaps?

**Comment by Chetan11-dev** [comment]
Could you explain more?
